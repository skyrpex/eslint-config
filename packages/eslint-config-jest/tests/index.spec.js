const { createEngine, cleanReport } = require("../../../tests/utils");

const engine = createEngine({
    configFile: require.resolve("../index.js"),
});

test("should report Jest errors", () => {
    const report = engine.executeOnFiles(["jest.fixture.js"]);

    expect(cleanReport(report)).toMatchSnapshot();
});
