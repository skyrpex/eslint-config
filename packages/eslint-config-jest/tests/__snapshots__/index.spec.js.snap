// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should report Jest errors 1`] = `
Object {
  "errorCount": 0,
  "fixableErrorCount": 0,
  "fixableWarningCount": 0,
  "results": Array [
    Object {
      "errorCount": 0,
      "filePath": "tests/fixtures/jest.fixture.js",
      "fixableErrorCount": 0,
      "fixableWarningCount": 0,
      "messages": Array [],
      "warningCount": 0,
    },
  ],
  "warningCount": 0,
}
`;
